lib/streamlit/git_util.py

```diff
@@ -107,3 +120,21 @@ def get_repo_info(self):
             return None
 
         return repo, branch, self.module
+
+    def get_uncommitted_files(self):
```

Comment:
Hm, is there a reason `untracked_file` and `is_head_detached` are properties, but `get_uncommited_files` and `get_ahead_commits` are not? Would prefer to make them all properties (eg `@property uncommitted files`, `ahead_commits`)

The other way (no properties) is probably okay too; I'm mostly looking for consistency.

Reply:
Yes, I don't have a strong opinion about this besides both `untracked_file` and `is_head_detached` are calling directly another property.

Reply:
Okay! All else being equal, I'd prefer to use properties, as it seems to be more Pythonic than using getters: https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters

Reply:
done!